### ISSUE TITLE
Comfort Keys Pro crap files

### ISSUE THREAD

**Issue by Shakil-Shahadat** [MoscaDotTo/Winapp2#827]
```
[Comfort Keys Pro *]
LangSecRef=3021
DetectFile=%AppData%\ComfortSoftware
FileKey1=%AppData%\ComfortSoftware\CKeys|*.*
FileKey2=%ProgramFiles%\ComfortKeys|changes.rtf;CKeys.chm;CS.url;License.rtf
```
The last entry will delete some unnecessary files.

**Comment by APMichael** [comment]
> ```
> FileKey2=%ProgramFiles%\ComfortKeys|changes.rtf;CKeys.chm;CS.url;License.rtf
> ```
> The last entry will delete some unnecessary files.

Please do not delete files like Authors, Changelog, Help, License and Readme. These are part of the application and not temporary "garbage".

Feel free to make an "License Text Files" entry in the Winapp3.ini for this.

**Comment by Shakil-Shahadat** [comment]
Ok, the rest of the entry should be good to go.
```
[Comfort Keys Pro *]
LangSecRef=3021
DetectFile=%AppData%\ComfortSoftware
FileKey1=%AppData%\ComfortSoftware\CKeys|*.*
```

**Comment by APMichael** [comment]
Included in #824.
